warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -1275,12 +1294,18 @@ protected ASTJexlScript processTree(final ASTJexlScript originalQueryTree, Shard
                 }
                 innerStopwatch.stop();
                 
-                innerStopwatch = timers.newStartedStopwatch("DefaultQueryPlanner - Expand pushing functions into exceeded value ranges");
-                queryTree = PushFunctionsIntoExceededValueRanges.pushFunctions(queryTree, metadataHelper, config.getDatatypeFilter());
-                if (log.isDebugEnabled()) {
-                    logQuery(queryTree, "Query after expanding pushing functions into exceeded value ranges:");
+                // The query tree was modified. Recount nodes.
+                nodeCount = NodeTypeCountVisitor.countNodes(queryTree);
```

Comment:
may not need this call as this would not affect the call at 1301.

Reply:
Removed.